src/confluent_kafka/src/Admin.c

```diff
@@ -2835,6 +3195,89 @@ static PyObject *Admin_c_DescribeConsumerGroupsResults_to_py(
+        cfl_PyDict_SetString(result, "user", rd_kafka_UserScramCredentialsDescription_user(description));
+
+        scramcredentialinfo_cnt = rd_kafka_UserScramCredentialsDescription_scramcredentialinfo_count(description);
+        scram_credential_infos = PyList_New(scramcredentialinfo_cnt);
+        for(i=0;i<scramcredentialinfo_cnt;i++){
+                PyList_SET_ITEM(scram_credential_infos,i,Admin_c_ScramCredentialInfo_to_py(rd_kafka_UserScramCredentialsDescription_scramcredentialinfo(description,i)));
+        }
+
+        PyDict_SetItemString(result,"scram_credential_infos",scram_credential_infos);
+        args = PyTuple_New(0);
+        result = PyObject_Call(cfl_PyObject_lookup("confluent_kafka.admin",
+                                                   "UserScramCredentialsDescription"), args, result);
+        return result;
+}
+
+static PyObject *Admin_c_DescribeUserScramCredentialsResult_to_py(const rd_kafka_DescribeUserScramCredentials_result_t *result_event) {
+        PyObject *result = NULL;
+        const rd_kafka_UserScramCredentialsDescription_t **descriptions;
+        size_t description_cnt;
+        size_t i;
+
+        result = PyDict_New();
+        descriptions = rd_kafka_DescribeUserScramCredentials_result_descriptions(result_event, &description_cnt);
+        for(i=0; i<description_cnt; i++){
+                const char *username;
+                const rd_kafka_error_t *error;
+                const rd_kafka_UserScramCredentialsDescription_t *description = descriptions[i];
+                username = rd_kafka_UserScramCredentialsDescription_user(description);
+                error = rd_kafka_UserScramCredentialsDescription_error(description);
+                if (rd_kafka_error_code(error)) {
+                        PyDict_SetItemString(result,username, KafkaError_new_or_None(rd_kafka_error_code(error),rd_kafka_error_string(error)));
+                } else {
+                        PyDict_SetItemString(result,username, Admin_c_UserScramCredentialsDescription_to_py(description));
+                }
+        }
+        return result;
+}
+
+static PyObject *Admin_c_AlterUserScramCredentialsResult_to_py(const rd_kafka_AlterUserScramCredentials_result_t *result_event) {
+        PyObject *result = NULL;
+        size_t cnt, i;
+        const rd_kafka_AlterUserScramCredentials_result_response_t **responses =
+                rd_kafka_AlterUserScramCredentials_result_responses(result_event, &cnt);
+        result = PyDict_New();
+        for(i=0; i<cnt; i++){
+                const rd_kafka_AlterUserScramCredentials_result_response_t *response = responses[i];
+                const rd_kafka_error_t *error = rd_kafka_AlterUserScramCredentials_result_response_error(response);
+                const char *username = rd_kafka_AlterUserScramCredentials_result_response_user(response);
+                PyDict_SetItemString(result, username, KafkaError_new_or_None(rd_kafka_error_code(error),rd_kafka_error_string(error)));
```

Comment:
Since (user, mechanism) is unique in the request, only the last one is kept here in the response. This is incorrect.